src/configs/strings.ts

```diff
@@ -2,4 +2,5 @@ export const GLOBAL_STRINGS = {
   unassignComment: "Releasing the bounty back to dev pool because the allocated duration already ended!",
   askUpdate: "Do you have any updates",
   assignCommandDisabledComment: "The `/assign` command is disabled for this repository.",
+  autopaycomment: "Autopay mode for this issue is set:",
```

Comment:
```suggestion
  autopaycomment: "Payment permit generation for this issue is enabled:",
```

It would be more elegant to write:
- "Payment permit generation for this issue is enabled."
- "Payment permit generation for this issue is disabled."